.editorconfig

```diff
@@ -200,9 +200,10 @@ dotnet_analyzer_diagnostic.category-Usage.severity            = warning
 
 dotnet_diagnostic.IDE0003.severity = error      # this and Me preferences
 dotnet_diagnostic.IDE0007.severity = error      # Use 'var' instead of explicit type
-dotnet_diagnostic.IDE0005.severity = error      # Remove unnecessary using directives
+dotnet_diagnostic.IDE0005.severity = warning    # Remove unnecessary using directives
```

Comment:
Let's keep this as an error too

Reply:
This should be fine with Joe's change.  We still have warnings-as-errors for everything but the local VS IDE build, so setting it to warning here should speed up the inner loop while still being errors before check-in.